src/game_updater/GameUpdater.cpp

```diff
@@ -134,6 +134,18 @@ void GameUpdater::Reset(std::shared_ptr<GameState> game_state) {
   }
 }
 
+bool GameUpdater::JumpInput() {
+  if (InputBindings::ButtonNewlyPressed(XBOX_A)) {
+    return true;
+  }
```

Comment:
I think there should be a permanent mapping of spacebar to A button in InputBindings instead of having this mapping here. Any time we check for a "spacebar" input, it should also check for the A button